src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/AzureSql/ContainerTests.cs

```diff
@@ -13,39 +13,44 @@
 // ----------------------------------------------------------------------------------
 
 using Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models;
-using Microsoft.Azure.ServiceManagement.Common.Models;
 using Microsoft.WindowsAzure.Commands.ScenarioTest;
 using Microsoft.WindowsAzure.Commands.Test.Utilities.Common;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
 using Xunit;
 
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Test.ScenarioTests
 {
-    public partial class ContainerTests : RMTestBase
+    public partial class ContainerTests : RecoveryServicesBackupTestRunner
     {
-        public XunitTracingInterceptor _logger;
+        private readonly string _commonModule1 = $"ScenarioTests/Common.ps1";
```

Comment:
DO NOT name with meaningless numbers. You can call it _containerCommModule.

Reply:
Some names of these files are duplicate, so I update them to folder name. Thanks.